src/main/index.ts

```diff
@@ -9,13 +8,24 @@ import { join } from "path";
 async function createWindow() {
   const [width, height] = getBounds();
 
+  const getIconPath = () => {
```

Comment:
This ternary operator mayhem should be rewritten. Ternary operators are fine. They become hard to work with when they are nested. Prefer standalone functions. Consider rewriting as:

```typescript
function getIcon(): string {
  if (process.platform === 'win32') {
    return join(__dirname, '../../build/icon.ico');
  }

  if (process.platform === 'darwin') {
    return join(__dirname, '../../build/icon.icns');
  }

  return join(__dirname, '../../build/icon.png');
}

Reply:
you can provide code as a suggestion btw if you select multiple lines and then press the first button in the toolbar or manually write
\`\`\`suggestion
\`\`\`

Reply:
My apologies on that I thought ternary operators looked cleaner lol. Should be changed to what you sent.